stable/nginx-ingress/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.37.0
+version: 1.37.1
```

We will need a minor version bump here as this is a new feature.
```suggestion
version: 1.38.0
```